src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,27 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(1, $card-size);
+  gap: 48px;
+  justify-content: center;
+
+  $media-querry:
+    2 488px,
+    3 768px,
+    4 1024px;
+
+  @each $number, $wide in $media-querry {
+    @media (min-width: $wide) {
+      grid-template-columns: repeat($number, $card-size);
+    }
+  }
```

Comment:
The use of this loop worsens the readability of the code. In order to fully comprehend what is happening, one must check the $media-query variable.